rs/sns/testing/BUILD.bazel

```diff
@@ -0,0 +1,92 @@
+    "//rs/nns/constants",
+    "//rs/rust_canisters/canister_test",
+    "@crate_index//:candid",
+    "@crate_index//:clap",
+    "@crate_index//:reqwest",
+    "@crate_index//:tokio",
+]
+
+MACRO_DEPENDENCIES = []
+
+ALIASES = {}
+
+DEV_DATA = [
+    "//rs/ledger_suite/icp/archive:ledger-archive-node-canister-wasm",
+    "//rs/ledger_suite/icp/index:ic-icp-index-canister",
+    "//rs/ledger_suite/icp/ledger:ledger-canister-wasm",
+    "//rs/ledger_suite/icp/ledger:ledger-canister-wasm-notify-method",
+    "//rs/ledger_suite/icrc1/archive:archive_canister",
+    "//rs/ledger_suite/icrc1/index-ng:index_ng_canister",
+    "//rs/ledger_suite/icrc1/ledger:ledger_canister",
+    "//rs/nns/cmc:cycles-minting-canister",
+    "//rs/nns/governance:governance-canister",
+    "//rs/nns/handlers/root/impl:root-canister",
+    "//rs/nns/sns-wasm:sns-wasm-canister",
+    "//rs/pocket_ic_server:pocket-ic-server",
+    "//rs/registry/canister:registry-canister",
+    "//rs/sns/governance:sns-governance-canister",
+    "//rs/sns/root:sns-root-canister",
+    "//rs/sns/swap:sns-swap-canister",
+    "@nns_dapp_canister//file",
+    "@sns_aggregator//file",
+    "@ii_dev_canister//file",
+]
+
+DEV_ENV = {
+    "CYCLES_MINTING_CANISTER_WASM_PATH": "$(rootpath //rs/nns/cmc:cycles-minting-canister)",
+    "GOVERNANCE_CANISTER_WASM_PATH": "$(rootpath //rs/nns/governance:governance-canister)",
+    "REGISTRY_CANISTER_WASM_PATH": "$(rootpath //rs/registry/canister:registry-canister)",
+    "IC_ICRC1_ARCHIVE_WASM_PATH": "$(rootpath //rs/ledger_suite/icrc1/archive:archive_canister)",
+    "IC_ICRC1_INDEX_NG_WASM_PATH": "$(rootpath //rs/ledger_suite/icrc1/index-ng:index_ng_canister)",
+    "IC_ICRC1_LEDGER_WASM_PATH": "$(rootpath //rs/ledger_suite/icrc1/ledger:ledger_canister)",
+    "IC_ICP_INDEX_WASM_PATH": "$(rootpath //rs/ledger_suite/icp/index:ic-icp-index-canister)",
+    "LEDGER_CANISTER_WASM_PATH": "$(rootpath //rs/ledger_suite/icp/ledger:ledger-canister-wasm)",
+    "LEDGER_CANISTER_NOTIFY_METHOD_WASM_PATH": "$(rootpath //rs/ledger_suite/icp/ledger:ledger-canister-wasm-notify-method)",
+    "LEDGER_ARCHIVE_NODE_CANISTER_WASM_PATH": "$(rootpath //rs/ledger_suite/icp/archive:ledger-archive-node-canister-wasm)",
+    "SNS_WASM_CANISTER_WASM_PATH": "$(rootpath //rs/nns/sns-wasm:sns-wasm-canister)",
+    "SNS_GOVERNANCE_CANISTER_WASM_PATH": "$(rootpath //rs/sns/governance:sns-governance-canister)",
+    "SNS_ROOT_CANISTER_WASM_PATH": "$(rootpath //rs/sns/root:sns-root-canister)",
+    "SNS_SWAP_CANISTER_WASM_PATH": "$(rootpath //rs/sns/swap:sns-swap-canister)",
+    "ROOT_CANISTER_WASM_PATH": "$(rootpath //rs/nns/handlers/root/impl:root-canister)",
+    "POCKET_IC_BIN": "$(rootpath //rs/pocket_ic_server:pocket-ic-server)",
+    "NNS_DAPP_WASM_PATH": "$(rootpath @nns_dapp_canister//file)",
+    "SNS_AGGREGATOR_WASM_PATH": "$(rootpath @sns_aggregator//file)",
+    "INTERNET_IDENTITY_WASM_PATH": "$(rootpath @ii_dev_canister//file)",
+}
+
+rust_binary(
```

Comment:
Could you add a doc comment to explain how to use this tool, please? 

Maybe give an example how the arguments can be passed, if called via Bazel.

Reply:
There is a [README](https://github.com/dfinity/ic/blob/018ddc30d0c5355e9c7cd5681c5eba173aba7de9/rs/sns/testing/README.md) in the root of this crate